be/src/agent/task_worker_pool.cpp

```diff
@@ -1185,9 +1185,15 @@ void TaskWorkerPool::_report_tablet_worker_thread_callback() {
 
         _is_doing_work = true;
         request.tablets.clear();
+        uint64_t report_version = _s_report_version;
         OLAPStatus build_all_report_tablets_info_status =
                 StorageEngine::instance()->tablet_manager()->build_all_report_tablets_info(
                         &request.tablets);
+        if (report_version < _s_report_version) {
```

Comment:
Add a TODO here to notice that this can only reduce the possibility or report error, but can't avoid it.